.github/CODEOWNERS

```diff
@@ -86,6 +86,8 @@
 
 /doc/api/modules.md @nodejs/modules
 /doc/api/esm.md @nodejs/modules
+/doc/api/module.md @nodejs/modules
+/doc/api/packages.md @nodejs/modules
```

Comment:
So this is consistent with the other unaltered lines but should this (and the others) be `@nodejs/modules-active-members`?

Reply:
I'm ok with it being @nodejs/modules as far as who it pings. The entire group is able to participate in consensus building / decision making... the active members is more about expectations for quorum and ability to vote